### DISCUSSION TITLE
Did you know? server feature

### DISCUSSION THREAD

**Discussion by Skaidus** [Glazelf/NinigiBot#129] [Ideas]
Basically something like stan command but with a database of curious/useless facts of the server that pop up randomly each day/each X days to keep people entertained. 

ie: Did you know first Shinx server message was from X user?

Maybe with YES/NO reactions, idk

**Comment by Glazelf** [comment]
Converted to a discussion so that the details can be discussed and set out here before we make an issue out of it
